### PR TITLE
Enable Circular Statistics module again

### PR DESCRIPTION
**Pull request by Kucharssim**
... and add circular example to Data Library

related to https://github.com/jasp-stats/jaspCircular/pull/2

### CODE REVIEW

File: Resources/Data Sets/index_nl.json [hunk 1]
```diff
@@ -828,8 +828,23 @@
...
-      "path": "16. Miscellaneous",
+      "name": "16. Circular Statistics",
+      "path": "16. Circular Statistics",
+      "description": "",
+      "kind": "folder",
+      "children": [
+        {
+          "name": "El Nino",
+          "path": "El Nino.jasp",
+          "description": "Wind direction measurements in Hawaii during the months August, July, and September. <br><br> The example JASP file demonstrates the use of circular descriptives and comparison of multiple circular variables. <br><br> <i>Fictional data form Aaron Bahde under CC-BY 4.0 license.<\/i>",
```
Comment (vandenman) [inline Resources/Data Sets/index_nl.json]:
```suggestion
          "description": "Metingen van de windrichting in Hawaii gedurende de maanden augustus, juli en september. <br><br> Het voorbeeld JASP-bestand demonstreert het gebruik van circulaire beschrijvingen en vergelijking van meerdere circulaire variabelen. <br><br> <i>Fictieve gegevens van Aaron Bahde onder CC-BY 4.0 licentie.<\/i>",
```

File: Resources/Data Sets/index.json [hunk 2]
```diff
@@ -828,8 +828,23 @@
...
-      "path": "16. Miscellaneous",
+      "name": "16. Circular Statistics",
+      "path": "16. Circular Statistics",
+      "description": "",
+      "kind": "folder",
+      "children": [
+        {
+          "name": "El Nino",
+          "path": "El Nino.jasp",
+          "description": "Wind direction measurements in Hawaii during the months August, July, and September. <br><br> The example JASP file demonstrates the use of circular descriptives and comparison of multiple circular variables. <br><br> <i>Fictional data form Aaron Bahde under CC-BY 4.0 license.<\/i>",
```
Comment (vandenman) [inline Resources/Data Sets/index.json]:
```suggestion
          "description": "Wind direction measurements in Hawaii during the months August, July, and September. <br><br> The example JASP file demonstrates the use of circular descriptives and comparison of multiple circular variables. <br><br> <i>Fictional data from Aaron Bahde under CC-BY 4.0 license.<\/i>",
```

File: Resources/Data Sets/index_de.json [hunk 3]
```diff
@@ -828,8 +828,23 @@
...
-      "path": "16. Miscellaneous",
+      "name": "16. Circular Statistics",
+      "path": "16. Circular Statistics",
+      "description": "",
+      "kind": "folder",
+      "children": [
+        {
+          "name": "El Nino",
+          "path": "El Nino.jasp",
+          "description": "Wind direction measurements in Hawaii during the months August, July, and September. <br><br> The example JASP file demonstrates the use of circular descriptives and comparison of multiple circular variables. <br><br> <i>Fictional data form Aaron Bahde under CC-BY 4.0 license.<\/i>",
```
Comment (vandenman) [inline Resources/Data Sets/index_de.json]:
```suggestion
          "description": "Wind direction measurements in Hawaii during the months August, July, and September. <br><br> The example JASP file demonstrates the use of circular descriptives and comparison of multiple circular variables. <br><br> <i>Fictional data from Aaron Bahde under CC-BY 4.0 license.<\/i>",
```

**Review by vandenman — COMMENTED** [review]
One typo and a suggestion for a Dutch translation.

**Review by AlexanderLyNL — APPROVED** [review]

**Pull request merged**